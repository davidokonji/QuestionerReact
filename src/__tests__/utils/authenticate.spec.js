import React from 'react';
import { shallow, mount } from 'enzyme';
import Protected from '../../utils/authenticate';

describe('Authenticate HOC', () => {
  it('should render HOC component correctly', () => {
    const wrapper = shallow(<Protected />);

    expect(wrapper).toMatchSnapshot();
  });
});
