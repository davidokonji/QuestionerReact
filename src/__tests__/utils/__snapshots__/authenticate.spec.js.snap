// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authenticate HOC should render HOC component correctly 1`] = `
<Route
  render={[Function]}
/>
`;
